src/NFCManager.cpp

```
@@ -2682,32 +2680,62 @@ bool NFCManager::isWriteQueueEmpty() const {
 
 bool NFCManager::readBambuTag(const uint8_t* uid, uint8_t uidLength, BambuTagData& out) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)